File: Web/kashish12_kashishalikhan_2125cse1133_02/pattern/pattern7.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;

    //write your code here
    for(int i = 1; i <= n; i++){
        for(int j = 1; j <= i; j++){
            if(i == j){
                cout<<"*\t";
                break;
            }
            else{
                cout<<"\t";
            }
        }
        cout<<endl;
    }
}

File: AR-VR/Samikshagp_SamikshaGupta_2125cse1177_2/Patterns/pattern7.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
int n=scn.nextInt();
int a[][]=new int [n][n];
for(int i=0;i<n;i++)
{
    for(int y=0;y<n;y++)
    {
        if((i+y)==n-1)
        {
            System.out.print("*");
        }
        else
        System.out.print(" ");
    }
    System.out.println();
}
        // write ur code here

    }
}